lab1/lab1/books/BasicBook.swift

```diff
@@ -0,0 +1,25 @@
+import Foundation
+
+public class BasicBook: LibraryItem {
+    public let title: String
+    
+    public let author: String
+    
+    public let publicationYear: Int
+    
+    public let genre: Genre
+    
+    public let id: UUID
```

Comment:
```suggestion
    public let id: UUID = .init()
```

И тогда не пришлось бы `init` писать, он бы синтезировался